packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5633,34 +5017,59 @@ export default class Meeting extends StatelessWebexPlugin {
       },
     });
 
-    return MeetingUtil.validateOptions(options)
-      .then(() => {
-        this.locusMediaRequest = new LocusMediaRequest(
-          {
-            correlationId: this.correlationId,
-            device: {
-              url: this.deviceUrl,
-              // @ts-ignore
-              deviceType: this.config.deviceType,
-            },
-            preferTranscoding: !this.isMultistream,
-          },
-          {
-            // @ts-ignore
-            parent: this.webex,
-          }
-        );
-      })
+    // we set sendAudio & sendVideo to true even before any tracks are published
```

Comment:
The comment suggests that it will always set sendAudio/sendVideo to true, but in reality it just defaults to true. It could still be passed in as false.